iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/plugin/TerraformSettings.java

```diff
@@ -45,8 +45,9 @@ public static List<PropertyDefinition> getProperties() {
       PropertyDefinition.builder(ACTIVATION_KEY)
         .index(1)
         .defaultValue(ACTIVATION_DEFAULT_VALUE)
-        .name("Activate Terraform analysis")
-        .description("Activate analysis of Terraform files.")
+        .name("Activate Terraform Analysis")
+        .description("Disabling Terraform analysis ensures that no Terraform files are parsed, highlighted and analyzed, " +
+          "and no IaC analysis results are included in the quality gate.")
```

Comment:
While taking a look at the documentation, I noticed that this description is no longer quite accurate, as analysis is now enabled by default.